esrally/config.py

```diff
@@ -527,7 +527,7 @@ def migrate(config_file, current_version, target_version, out=print, i=input):
         raise ConfigError("The config file in {} is too old. Please delete it and reconfigure Rally from scratch with {} configure."
                           .format(config_file.location, PROGRAM_NAME))
 
-    prompter = Prompter(i=i, o=out, assume_defaults=False)
+    Prompter(i=i, o=out, assume_defaults=False)
```

Comment:
How about we remove the entire statement as it does not have any effect?

Reply:
I was afraid of unintended side affects here, so I did not remove. But Ill assume you know that it doesnt do something magic and remove :)